.github/workflows/gh-pages.yml

```
@@ -204,31 +204,42 @@ jobs:
             fonts
 
+      - name: Cache Woff2 output
+        uses: actions/cache@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4
+        with:
+          path: fonts/woff2
+          key: python-${{ runner.os }}-fonts-${{ hashFiles('fonts/**') }}
+          restore-keys: |
+            python-${{ runner.os }}-fonts-
+
       - name: Install Poetry
+        if: steps.cache.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "cache" is not defined in object type {}

(expression)